wpilibcExamples/src/main/cpp/examples/GearsBot/include/subsystems/Wrist.h

```diff
@@ -41,14 +42,12 @@ class Wrist : public frc2::PIDSubsystem {
 
  private:
   frc::PWMSparkMax m_motor{6};
-
-// Conversion value of potentiometer varies between the real world and
-// simulation
-#ifndef SIMULATION
-  frc::AnalogPotentiometer m_pot{3, -270.0 / 5};
-#else
-  frc::AnalogPotentiometer m_pot{3};  // Defaults to degrees
-#endif
+  // Conversion value of potentiometer varies between the real world and
+  // simulation
+  frc::AnalogPotentiometer m_pot =
+      frc::RobotBase::IsReal()
```

Comment:
Everywhere else uses IsSimulation() except here:
```bash
wpilibc/src/main/native/cpp/PneumaticHub.cpp
63:    if (version.FirmwareMajor > 0 && RobotBase::IsReal()) {
```

Reply:
Does it matter? 
Also, the java example uses isReal: https://github.com/wpilibsuite/allwpilib/blob/685c73256881d61c046c4c8c3106df21e483a9ce/wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/examples/gearsbot/subsystems/Wrist.java#L29

Reply:
:shrug: it's a consistency thing.

Reply:
Search results (all in `**/src` directories, on 4226d11):
```
IsReal():
  wpilibc/src/main/native/include/frc/RobotBase.h:219:  static constexpr bool IsReal() {
  wpilibc/src/main/native/cpp/PneumaticHub.cpp:63:    if (version.FirmwareMajor > 0 && RobotBase::IsReal()) {
IsSimulation():
  wpilibNewCommands/src/main/native/cpp/frc2/command/CommandScheduler.cpp:183:    if constexpr (frc::RobotBase::IsSimulation()) {
  wpilibc/src/main/native/cppcs/RobotBase.cpp:233:  if constexpr (!IsSimulation()) {
  wpilibc/src/main/native/cppcs/RobotBase.cpp:253:  if constexpr (!IsSimulation()) {
  wpilibc/src/main/native/include/frc/RobotBase.h:232:  static constexpr bool IsSimulation() {
  wpilibc/src/main/native/cpp/Filesystem.cpp:16:  if constexpr (!RobotBase::IsSimulation()) {
  wpilibc/src/main/native/cpp/Filesystem.cpp:24:  if constexpr (!RobotBase::IsSimulation()) {
  wpilibc/src/main/native/cpp/TimedRobot.cpp:23:  if constexpr (IsSimulation()) {
isReal():
  wpilibj/src/main/java/edu/wpi/first/wpilibj/RobotBase.java:202:  public static boolean isReal() {
  wpilibj/src/main/java/edu/wpi/first/wpilibj/PneumaticHub.java:40:      if (version.firmwareMajor > 0 && RobotBase.isReal()) {
  wpilibj/src/main/java/edu/wpi/first/wpilibj/DataLogManager.java:213:    if (RobotBase.isReal()) {
  wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/examples/gearsbot/subsystems/Drivetrain.java:62:    if (Robot.isReal()) {
  wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/examples/gearsbot/subsystems/Wrist.java:29:    if (Robot.isReal()) {
  wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/examples/gearsbot/subsystems/Elevator.java:41:    if (Robot.isReal()) {
isSimulation():
  wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/CommandScheduler.java:264:      if (RobotBase.isSimulation()) {
  wpilibj/src/main/java/edu/wpi/first/wpilibj/Filesystem.java:42:    if (!RobotBase.isSimulation()) {
  wpilibj/src/main/java/edu/wpi/first/wpilibj/Filesystem.java:57:    if (!RobotBase.isSimulation()) {
  wpilibj/src/main/java/edu/wpi/first/wpilibj/RobotBase.java:76:            return !RobotBase.isSimulation();
  wpilibj/src/main/java/edu/wpi/first/wpilibj/RobotBase.java:141:    if (!isSimulation()) {
  wpilibj/src/main/java/edu/wpi/first/wpilibj/RobotBase.java:193:  public static boolean isSimulation() {
  wpilibj/src/main/java/edu/wpi/first/wpilibj/RobotBase.java:328:    if (!isSimulation()) {
  wpilibj/src/main/java/edu/wpi/first/wpilibj/IterativeRobotBase.java:408:    if (isSimulation()) {
  wpilibj/src/main/java/edu/wpi/first/wpilibj/TimedRobot.java:110:    if (isSimulation()) {
  wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/examples/gearsbot/subsystems/Elevator.java:29:    if (Robot.isSimulation()) { // Check for simulation and update PID values
  wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/examples/gearsbot/commands/CloseClaw.java:39:    if (!Robot.isSimulation()) {
```